website/docs/r/app_service_connection.html.markdown

```diff
@@ -112,6 +112,34 @@ An `authentication` block supports the following:
 
 ---
 
+* `scope` - (Optional) Connection scope in source service. It's namespace name if source is AKS cluster, container name if source is Container app.
```

Comment:
I think this reads more clearly for the user grammatically. 
```suggestion
* `scope` - (Optional) Connection scope in the source service. This is the Namespace Name if the source is an AKS cluster, or the Container Name if source is a Container App.
```

Reply:
Sure, will fix.